src/text-buffer.coffee

```diff
@@ -18,8 +19,11 @@ class TextBuffer
   getLinesLayer: ->
     @linesLayer ?= new Layer(@getCharactersLayer(), new LinesTransform)
 
-  buildTabsLayer: (tabLength) ->
-    new Layer(@getLinesLayer(), new TabsTransform(tabLength))
+  buildHardTabsLayer: (tabLength) ->
+    new Layer(@getLinesLayer(), new HardTabsTransform(tabLength))
```

Comment:
Agreed :+1: My concern was more about whether extracting this now or later on. I am not entirely sure that removing duplication, like in this case, is beneficial in these early stages: since this code is subject to change, I do not have enough confidence to extract it right now. My proposal would be to wait for extraction until either:

1. Changing one requires to change the other
2. We add a third transformation, which resembles the previous two

Otherwise, if you think it's fine to apply that refactoring now, I am fine to do that as well :blush: